client/header/activity-panel/wordpress-notices.js

```diff
@@ -135,6 +135,12 @@ class WordPressNotices extends Component {
 		if ( 'message' === element.id && 'updated' === element.className ) {
 			return false;
 		}
+
+		if ( 'dolly' === element.id ) {
```

Comment:
Optional refactor:  Since this list of notices is growing and may continue to grow, I'm wondering if we should create an array of items with IDs and class names.

```js
const noticesToHide = [
    [ 'message', [ 'notice' ] ], // ID and array of classes
    [ 'dolly', null ],
]
```
Then again, this may not be ideal if we need to apply CSS styles here or target anything outside the realm of IDs and classes.

Reply:
Decided to give your suggest a try: https://github.com/woocommerce/wc-admin/pull/1560/commits/c979c9d1049e6cc8459e14f8394f142e78bebf50 

Makes code a bit more lengthy but makes it easy to add new ids/classes.

Happy to revert it though!